{
	"id": "115278",
	"key": "AC-2722",
	"fields": {
		"issuetype": {
			"id": "1",
			"description": "A problem which impairs or prevents the functions of the product.",
			"name": "Bug",
			"subtask": false
		},
		"project": {
			"id": "12217",
			"key": "AC",
			"name": "Appcelerator - INBOX",
			"projectCategory": {
				"id": "10000",
				"description": "",
				"name": "Customer Service"
			}
		},
		"resolution": {
			"id": "5",
			"description": "All attempts at reproducing this issue failed, or not enough information was available to reproduce the issue. Reading the code produces no clues as to why this behavior would occur. If more information appears later, please reopen the issue.",
			"name": "Cannot Reproduce"
		},
		"resolutiondate": "2013-05-31T19:22:47.000+0000",
		"created": "2013-05-31T14:18:13.000+0000",
		"labels": [
			"ios"
		],
		"versions": [],
		"issuelinks": [],
		"assignee": {
			"name": "mpmiranda",
			"key": "mpmiranda",
			"displayName": "Mauro Parra-Miranda",
			"active": true,
			"timeZone": "America/Mexico_City"
		},
		"updated": "2016-03-08T07:41:50.000+0000",
		"status": {
			"description": "The issue is considered finished, the resolution is correct. Issues which are closed can be reopened.",
			"name": "Closed",
			"id": "6",
			"statusCategory": {
				"id": 3,
				"key": "done",
				"colorName": "green",
				"name": "Done"
			}
		},
		"components": [
			{
				"id": "14550",
				"name": "Appcelerator Modules",
				"description": "Please enter tickets related to Modules here."
			},
			{
				"id": "14551",
				"name": "Studio",
				"description": "Please enter tickets related to Titanium Studio here."
			}
		],
		"description": "I create a module in Titanium Studio, build: 3.1.0.201304151600 for 2.1.4 and when I include it in an ios project I get the following:\r\n[WARN] Could not find Titanium Module id: org.xxx.av3titanium214, version: 3.0, platform: iphone [ERROR] Could not find the following required iOS modules: [ERROR] id: org.xxx.av3titanium214 version: 3.0 [ERROR] Error: Traceback (most recent call last): File \"/Library/Application Support/Titanium/mobilesdk/osx/2.1.4.GA/iphone/builder.py\", line 771, in main exit(1) File \"/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site.py\", line 364, in call raise SystemExit(code) SystemExit: 1\r\nThe module is in: /Library/Application Support/Titanium/modules/iphone/org.xxx.av3titanium214/3.0\r\nIf I create a 3.1 module it works fine.",
		"attachment": [],
		"flagged": false,
		"summary": "in Titanium Studio, build: 3.1.0.201304151600 when I create a 2.1.4 module apps cannot find the module",
		"creator": {
			"name": "jyoungapp",
			"key": "jyoungapp",
			"displayName": "Jim Young",
			"active": true,
			"timeZone": "America/Los_Angeles"
		},
		"subtasks": [],
		"reporter": {
			"name": "jyoungapp",
			"key": "jyoungapp",
			"displayName": "Jim Young",
			"active": true,
			"timeZone": "America/Los_Angeles"
		},
		"environment": "build: 3.1.0.201304151600",
		"comment": {
			"comments": [
				{
					"id": "255237",
					"author": {
						"name": "clathrop",
						"key": "clathrop",
						"displayName": "Carter Lathrop",
						"active": true,
						"timeZone": "America/Los_Angeles"
					},
					"body": "Jim, \n\nI was unable to reproduce this bug. Can you please provide a sample module that one can test on 2.1.4GA and also on 3.1GA to show the disparity. Also it would be helpful to provide a list of steps to reproduce and expected behavior vs actual behavior as outlined here: https://wiki.appcelerator.org/display/guides/How+to+Submit+a+Bug+Report\n\nMarking as cannot reproduce until more information provided.\n\nThanks,\nCarter",
					"updateAuthor": {
						"name": "clathrop",
						"key": "clathrop",
						"displayName": "Carter Lathrop",
						"active": true,
						"timeZone": "America/Los_Angeles"
					},
					"created": "2013-05-31T19:22:25.000+0000",
					"updated": "2013-05-31T19:22:25.000+0000"
				},
				{
					"id": "256100",
					"author": {
						"name": "jyoungapp",
						"key": "jyoungapp",
						"displayName": "Jim Young",
						"active": true,
						"timeZone": "America/Los_Angeles"
					},
					"body": "More Specifically:\r\n\r\nI created a new module setting the SDk to 2.1.4\r\n\r\nBuilt it and added it to my system, did not modify it at all!\r\n\r\nCreated a new mobile project setting SDK to 2.1.4 and copied the lines from the example app.js to require the module and print it out to my new mobile project.\r\n\r\n// TODO: write your module tests here\r\nvar wtf214 = require('org.myactions.wtf214');\r\nTi.API.info(\"module is => \" + wtf214);\r\n\r\nI added the module to the Modules area of tiApp.xml and got the following when I tried to run the simulator:\r\n\r\n[INFO] One moment, building ...\r\n[INFO] Detected third-party module: org.myactions.wtf214/0.1\r\n[INFO] Detected third-party module: org.myactions.wtf214/0.1\r\n[INFO] Titanium SDK version: 2.1.4 (11/09/12 12:46 51f2c64)\r\n[INFO] iPhone Device family: universal\r\n[INFO] iPhone SDK version: 6.1\r\n[INFO] iPhone simulated device: iphone\r\n[INFO] Minimum iOS version: 4.3 Linked iOS Version 6.1\r\n[INFO] Compiling localization files\r\n[INFO] Launching application in Simulator\r\n[INFO] Launched application in Simulator (16.26 seconds)\r\n[INFO] Application started\r\n[INFO] wtf214proj/1.0 (2.1.4.GA.51f2c64)\r\n[ERROR] Script Error = Couldn't find module: org.myactions.wtf214.\r\n\r\n\r\nSo I gave up and tried it using SDK 3.0.0 and it could find the module.\r\n\r\nHope this helps.\r\n\r\nJim Young",
					"updateAuthor": {
						"name": "jyoungapp",
						"key": "jyoungapp",
						"displayName": "Jim Young",
						"active": true,
						"timeZone": "America/Los_Angeles"
					},
					"created": "2013-06-05T13:13:19.000+0000",
					"updated": "2013-06-05T13:13:19.000+0000"
				},
				{
					"id": "281077",
					"author": {
						"name": "mpmiranda",
						"key": "mpmiranda",
						"displayName": "Mauro Parra-Miranda",
						"active": true,
						"timeZone": "America/Mexico_City"
					},
					"body": "Working in latest GA. ",
					"updateAuthor": {
						"name": "mpmiranda",
						"key": "mpmiranda",
						"displayName": "Mauro Parra-Miranda",
						"active": true,
						"timeZone": "America/Mexico_City"
					},
					"created": "2013-11-24T22:47:57.000+0000",
					"updated": "2013-11-24T22:47:57.000+0000"
				}
			],
			"maxResults": 3,
			"total": 3,
			"startAt": 0
		}
	}
}